Silly question, but how does one use this within nodejs?

```
var plugins = require('leapjs-plugins')
.
.
.
  var controller = new Leap.Controller(leapConfig);
  controller.use(plugins.handEntry)
  controller.on('handLost',
    function(hand) {
      console.log('hand lost', hand);
    }
  )
```
Blows up with:
```
Error: Cannot find module './hand-entry/leap.hand-entry.js'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/home/henry74/test/node_modules/leapjs-plugins/index.js:2:1)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
Program node index.js exited with code 8
```

--- a/CONTRIBUTORS
+++ b/CONTRIBUTORS
@@ -132,3 +132,7 @@ Email: pmandel@leapmotion.com
 4. Robin Böhm
 Github account: robinboehm
 Email: robinboehm@gmail.com
+
+5. Martin Naumann
+Github account: AVGP
+Email: martin@geekonaut.de

--- a/index.js
+++ b/index.js
@@ -1,3 +1,8 @@
 Leap = require('leapjs')
-require('./hand-entry/leap.hand-entry.js')
+require('./main/hand-entry/leap.hand-entry.js')
+require('./main/hand-hold/leap.hand-hold.js')
+require('./main/bone-hand/leap.bone-hand.js')
+require('./main/playback/leap.playback-0.2.1.js')
+require('./main/screen-position/leap.screen-position.js')
+require('./main/transform/leap.transform.js')
 module.exports = true

--- a/main/bone-hand/leap.bone-hand.coffee
+++ b/main/bone-hand/leap.bone-hand.coffee
@@ -1,5 +1,7 @@
 scope = null
 
+if require then THREE = require('three')
+
 initScene = (targetEl, scale)->
   scope.scene = new THREE.Scene()
   scope.rendererOps ||= {}

--- a/main/bone-hand/leap.bone-hand.js
+++ b/main/bone-hand/leap.bone-hand.js
@@ -1,8 +1,10 @@
 //CoffeeScript generated from main/bone-hand/leap.bone-hand.coffee
 (function() {
-  var HandMesh, armTopAndBottomRotation, baseBoneRotation, boneColor, boneHandLost, boneRadius, boneScale, initScene, jointColor, jointRadius, jointScale, material, onHand, scope;
+  var HandMesh, armTopAndBottomRotation, baseBoneRotation, boneColor, boneHandLost, boneRadius, boneScale, initScene, jointColor, jointRadius, jointScale, material, onHand, scope, THREE;
 
   scope = null;
+
+  if(require) THREE = require('three');
 
   initScene = function(targetEl, scale) {
     var camera, directionalLight, far, height, near, renderer, width;

--- a/package.json
+++ b/package.json
@@ -3,7 +3,9 @@
   "version": "0.1.11pre",
   "description": "A collection of useful plugins for LeapJS",
   "main": "main/leapjs-plugins-0.7.js",
-  "dependencies": {},
+  "dependencies": {
+    "three": "^0.69.0"
+  },
   "devDependencies": {
     "coffee-script": ">=1.7.1",
     "grunt-contrib-concat": "~0.3.0",

